src/Api/Response/Parser/FiltersParser.php

```diff
@@ -191,6 +196,12 @@ protected function parseFilter($filter, $isMainFilter = false)
             $filterData['step'] = (float)$filter->attributes->stepSize;
         }
 
+        if ($filterName === 'price' && $filterData['findologicFilterType'] === Plugin::FILTER_TYPE_RANGE_SLIDER) {
```

Comment:
Let's use this configuration for all range sliders.

```suggestion
        if ($filterData['findologicFilterType'] === Plugin::FILTER_TYPE_RANGE_SLIDER) {
```